src/main/java/seedu/address/autocomplete/UserinputParserUtil.java

```diff
@@ -0,0 +1,52 @@
+package seedu.address.autocomplete;
+
+import java.util.LinkedList;
+
+/**
+ * Utility class for parsing userinput
+ */
+public class UserinputParserUtil {
+
+    /**
+     * Parse userinput by spaces
+     *
+     * @param currentPhraseInCommandBox string in command box text field
+     * @return array of split strings
+     */
+    public static String[] splitIntoSegments(String currentPhraseInCommandBox) {
+        return currentPhraseInCommandBox.split(" ");
+    }
+
+    /**
+     * Parse first segment of userinput into object and command word
```

Comment:
what is a segment?

Reply:
A segment is from the start of userinput to the first space(ie visit-now-show). Will clarify that in the documentation.